cvat-ui/src/actions/settings-actions.ts

```diff
@@ -45,6 +47,9 @@ export enum SettingsActionTypes {
     SWITCH_TOOLS_BLOCKER_STATE = 'SWITCH_TOOLS_BLOCKER_STATE',
     SWITCH_SHOWING_DELETED_FRAMES = 'SWITCH_SHOWING_DELETED_FRAMES',
     SWITCH_SHOWING_TAGS_ON_FRAME = 'SWITCH_SHOWING_TAGS_ON_FRAME',
+    ADD_IMAGE_FILTER = 'ADD_IMAGE_FILTER',
+    REMOVE_IMAGE_FILTER = 'REMOVE_IMAGE_FILTER',
+    SETUP_IMAGE_FILTER = 'SETUP_IMAGE_FILTER',
```

Comment:
- ENABLE_IMAGE_FILTER
- DISABLE_IMAGE_FILTER

I fill like SETUP_IMAGE_FILTER is extra action, ENABLE with options is enough, just need to replace filter options in state. 

Instead of ``filterAlias: null`` in ``removeImageFilter`` I would propose to add action  RESET_IMAGE_FILTERS (it looks more natural then null argument)